### PR TITLE
@mzikherman: Some artwork Q/A items

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

File: schema/artist/index.js [hunk 1]
```diff
@@ -105,22 +106,14 @@ const ArtistType = new GraphQLObjectType({
...
-              resolve: ({ forsale_artworks_count }) => forsale_artworks_count,
-            },
+            artworks: numeral(({ published_artworks_count }) =>
+              published_artworks_count),
+            follows: numeral(({ follow_count }) =>
+              follow_count),
+            auction_lots: numeral(({ auction_lots_count }) =>
+              auction_lots_count),
+            for_sale_artworks: numeral(({ forsale_artworks_count }) =>
+              forsale_artworks_count),
```
Comment (dzucconi) [inline schema/artist/index.js]: For

Comment (mzikherman) [inline schema/artist/index.js]: nice

File: schema/artwork/context.js [hunk 2]
```diff
@@ -39,9 +39,9 @@ export default {
   resolve: ({ id }) =>
     Promise
       .all([
-        gravity('related/fairs', { artwork: [id], active: true, size: 1 }),
-        gravity('related/sales', { artwork: [id], active: true, size: 1 }),
-        gravity('related/shows', { artwork: [id], active: true, size: 1 }),
+        gravity('related/fairs', { artwork: [id], size: 1 }),
+        gravity('related/sales', { artwork: [id], size: 1 }),
```
Comment (mzikherman) [inline schema/artwork/context.js]: should this be `is_auction: true`? Also, if we want just 'live' auctions, can do `live: true`

Comment (dzucconi) [inline schema/artwork/context.js]: Actually no; believe it or not there are still normal sales. https://www.artsy.net/artwork/andy-warhol-electric-chair-18

**Pull request merged**